frontend/src/views/web/profile.html

```diff
@@ -2,23 +2,19 @@
     <slide-profile></slide-profile>
     <div class="ev-md-container center">
         <img ng-src="{{profile.imgUrl}}" height="80" width="auto">
-        <div>{{web.user.username}}</div>
-        <div class="text-light-gray	w-300">{{web.user.email}}</div>
+        <div>{{profile.user.username}}</div>
+        <div class="text-light-gray w-300">{{profile.user.email}}</div>
         <ul>
             <li>
-                <span class="text-light-gray fs-12">First Name<br> </span> {{web.user.first_name}}
+                <span class="text-light-gray fs-12">First Name<br> </span> {{profile.user.first_name}}
             </li>
             <li>
-                <span class="text-light-gray fs-12">Last Name<br></span> {{web.user.last_name}}
+                <span class="text-light-gray fs-12">Last Name<br></span> {{profile.user.last_name}}
             </li>
             <li>
                 <span class="text-light-gray fs-12">Affiliated To<br></span> -
             </li>
-            <li>
```

Comment:
@RishabhJain2018 make this as a comment. May be in near future we need this feature.